### PR TITLE
OAT Simulation Client

### PR DESCRIPTION
**Pull request by christian-kardach**
First iteration of the OAT Simulation client. Overall review would be nice when time allows!

### CODE REVIEW

File: OATControl/ViewModels/MountVM.cs [hunk 1]
```diff
@@ -2817,6 +2833,32 @@ public bool KeepMiniControlOnTop
 				}
 			}
 		}
+
+		private void UpdateSimulationClient()
+        {
+			if (_oatSimComm != null)
+			{
+				_oatSimComm.Send($"CurrentRAString|{CurrentRAHour},{CurrentRAMinute},{CurrentRASecond}");
```
Comment (ClutchplateDude) [inline OATControl/ViewModels/MountVM.cs:3005]: I would like to see this function split up into two. On connecting, send the non-dynamic properties (Version, FW version, Microsteps, etc.) and only the dynamic parts sent here. This function runs regularly, so we want it to be quick. Thoughts?

Comment (christian-kardach) [reply OATControl/ViewModels/MountVM.cs:3005]: Yes agree with that, will revise!

**Comment by christian-kardach** [comment]
So, Stellarium direct connection will have to wait so this PR is up for review when there's time!

File: OATSimulation/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("OATSimulation")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("OATSimulation")]
+[assembly: AssemblyCopyright("Copyright ©  2021")]
```
Comment (ClutchplateDude) [inline OATSimulation/Properties/AssemblyInfo.cs]: You should put "Copyright  c 2021 by OpenAstroTech" here. Or your name.

File: OATSimulation/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("OATSimulation")]
+[assembly: AssemblyDescription("")]
```
Comment (ClutchplateDude) [inline OATSimulation/Properties/AssemblyInfo.cs]: Description would be good to fill out

**Review by ClutchplateDude — APPROVED** [review]
Just some minor string settings, then it's good.

**Pull request merged**